scenarios/appsec/test_customconf.py

```diff
@@ -38,7 +38,7 @@ def test_c05(self):
         stdout.assert_presence(r"AppSec could not read the rule file .* as it was invalid: .*", level="CRITICAL")
 
 
-@released(java="0.93.0", nodejs="2.4.1", php_appsec="0.3.0", ruby="?")
+@released(java="0.93.0", nodejs="?", php_appsec="0.3.0", ruby="?")
```

Comment:
You deactivated the test, it's on purpose ?

Reply:
Yes, it's not implemented